src/client/localplayer.cpp

```diff
@@ -609,7 +610,8 @@ void LocalPlayer::applyControl(float dtime, Environment *env)
 				at its starting value
 			*/
 			v3f speedJ = getSpeed();
-			if (speedJ.Y >= -0.5f * BS) {
+			if ((!physics_override_new_jump && speedJ.Y >= -0.5f * BS) ||
+				(physics_override_new_jump && touching_ground && speedJ.Y >= -0.5f * BS && speedJ.Y <= 0.5f * BS)) {
```

Comment:
`fabsf(speedJ.Y) <= 0.5f * BS` would be shorter for the speed check

Reply:
Fixed